src/m365/flow/commands/flow-get.ts

```diff
@@ -17,6 +17,16 @@ interface Options extends GlobalOptions {
   asAdmin: boolean;
 }
 
+interface Triggers {
+  type: string;
+  kind?: string;
+}
+
+interface Actions {
```

Comment:
I suppose this interface describes a single action and should be named `Action`, right?